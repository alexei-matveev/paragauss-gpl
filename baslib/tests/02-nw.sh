../bin/qm-find-basis --library=nwchem fe 6-31g ahlrichs_coulomb_fitting
