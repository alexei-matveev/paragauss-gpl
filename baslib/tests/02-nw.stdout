&unique_atom_basisset
lmax_ob=2
lmax_ch=4
lmax_xc=0
lmax_pseudo=-1
zc=0
/
&unique_atom_basis
n_exponents=22
n_contracted_fcts=5
n_uncontracted_fcts=0
/
0.041148 0.114177 0.504091 0.736721 1.273643 1.950316 3.05342 6.066075 7.192086 14.72996 16.83929 36.3102 50.43485 64.44629 96.84917 188.7543 299.8761 589.2479 1259.98 2090.857 9179.342 61132.62
0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.3501976 0.0 0.4797058 0.0 0.2314823 0.0 0.06673128 0.01353038 0.001766111
0.0 0.0 0.0 0.0 0.0 0.0 0.0 -0.4502914 0.0 -0.6207593 0.0 -0.03139902 0.0 0.0 0.1265724 0.0 0.03224048 0.0 0.002438014 0.0 0.0 0.0
0.0 0.0 -0.2049353 0.0 -0.7342822 0.0 -0.2882837 0.0 0.2556591 0.0 0.07196598 0.0 -0.003873256 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0
0.0 -1.138159 0.0 0.2882915 0.0 0.05694869 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0
1.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0
&unique_atom_basis
n_exponents=16
n_contracted_fcts=4
n_uncontracted_fcts=0
/
0.041148 0.114177 0.504091 0.736721 1.273643 1.950316 3.05342 6.066075 7.192086 14.72996 16.83929 36.3102 50.43485 96.84917 299.8761 1259.98
0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.2043058 0.0 0.4617931 0.0 0.3487236 0.0 0.1368317 0.03144647 0.004028019
0.0 0.0 0.1791251 0.0 0.4989059 0.0 0.3954946 0.0 0.06181383 0.0 -0.0287766 0.0 -0.007017128 0.0 0.0 0.0
0.0 0.9076485 0.0 0.2852139 0.0 -0.4593796 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0
1.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0
&unique_atom_basis
n_exponents=4
n_contracted_fcts=2
n_uncontracted_fcts=0
/
0.504361 1.846601 6.122368 23.14994
0.0 0.7014816 0.3896319 0.08876935
1.0 0.0 0.0 0.0
&unique_atom_basis
n_exponents=0
n_contracted_fcts=0
n_uncontracted_fcts=0
/
&unique_atom_basis
n_exponents=16
n_contracted_fcts=6
n_uncontracted_fcts=0
/
0.098974 0.183693 0.341923 0.641992 1.222744 2.3751 4.728977 9.696774 20.564393 45.278613 103.853491 248.866093 624.601709 1645.24364 4555.93194 13280.7469
0.0 0.0 0.0 0.0 0.0 0.0 2.29526551 -5.52028999 15.4918296 13.8443326 -5.27093265 12.599002 24.8296595 15.5768087 5.3913269 2.49415899
0.0 0.0 0.0 0.0 1.23245923 4.11618242 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0
0.0 0.0 0.0 1.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0
0.0 0.0 1.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0
0.0 1.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0
1.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0
&unique_atom_basis
n_exponents=4
n_contracted_fcts=4
n_uncontracted_fcts=0
/
0.194938371 0.467852091 1.15428468 3.08055495
0.0 0.0 0.0 1.0
0.0 0.0 1.0 0.0
0.0 1.0 0.0 0.0
1.0 0.0 0.0 0.0
&unique_atom_basis
n_exponents=4
n_contracted_fcts=2
n_uncontracted_fcts=0
/
0.69321 1.672439 4.063616 10.082702
0.0 0.06042159 0.08387445 0.05113178
1.0 0.0 0.0 0.0
&unique_atom_basis
n_exponents=3
n_contracted_fcts=3
n_uncontracted_fcts=0
/
0.730487863 1.82621966 4.74817111
1.0 0.0 0.0
0.0 1.0 0.0
0.0 0.0 1.0
&unique_atom_basis
n_exponents=4
n_contracted_fcts=2
n_uncontracted_fcts=0
/
0.84558 2.26352 6.210359 18.283549
0.0 0.07957566 0.1425065 0.08353784
1.0 0.0 0.0 0.0
&unique_atom_basis
n_exponents=0
n_contracted_fcts=0
n_uncontracted_fcts=0
/
&unique_atom_basis
n_exponents=0
n_contracted_fcts=0
n_uncontracted_fcts=0
/
